package.json

```diff
@@ -27,6 +27,7 @@
     "@fortawesome/fontawesome-svg-core": "6.3.0",
     "@fortawesome/free-solid-svg-icons": "6.3.0",
     "@fortawesome/react-fontawesome": "0.2.0",
+    "@headlessui/react": "1.7.14",
```

Comment:
This library has a great react component for transition, and great components to use for dropdowns, modals, etc that are designed for tailwind. We can replace some of the other libraries I've added with this one in a future PR.

Reply:
I took a look at it's site. It looks pretty nice!